README.md

```diff
@@ -9,20 +9,24 @@ See also
 Installation
 ------------
 
-    curl https://raw.github.com/jistr/ansible-dconf/master/dconf.py > ~/ansible_dir/library/dconf
+    curl https://raw.githubusercontent.com/Leryan/ansible-dconf/master/dconf.py > ~/ansible_dir/library/dconf
 
 Usage examples
 --------------
 
     - name: gnome-terminal default-show-menubar
-      dconf: user=jistr
-             key=/org/gnome/terminal/legacy/default-show-menubar
-             value=false
+      dconf:
+        user: 'jistr'
+        key: '/org/gnome/terminal/legacy/default-show-menubar'
+        value: 'false'
+
+The `user` parameter is optional, using the currently connected user. If set, it uses `su - <user>` to switch.
```

Comment:
Yes, english isn't my primary language and i still struggle to keep sentences clear, no problem with that!